packages/core/core/src/AssetGraph.js

```diff
@@ -281,6 +285,23 @@ export default class AssetGraph extends Graph<AssetGraphNode>
     }, startNode);
   }
 
+  traverseAssetsWithReferences(
```

Comment:
hmmm... wondering about adding a whole new method for this. Any reason to not just use `traverse` directly?

Reply:
This is used in the packager, which we don't want access to the raw graph (see the public api discussion).

I could maybe see an option in `traverseAssets`, but it would overload the signature too much imo.

Reply:
One thing we could do is expose the more limited `traverse`[0] we have on the main graph on the bundles themselves, and add `asset_reference`s as well. That would address this nicely.

[0] https://github.com/parcel-bundler/parcel/pull/2892/files#diff-2ae149d0ae6e33d140f738e72ea8e8b1R54